src/commands/build/wranglerjs/mod.rs

```diff
@@ -39,26 +47,84 @@ pub fn run_build(project: &Project) -> Result<(), failure::Error> {
         let wranglerjs_output: WranglerjsOutput =
             serde_json::from_str(&output).expect("could not parse wranglerjs output");
 
-        if wranglerjs_output.has_errors() {
-            message::user_error(wranglerjs_output.get_errors().as_str());
-            failure::bail!("Webpack returned an error");
-        }
+        write_wranglerjs_output(&bundle, &wranglerjs_output)
+    } else {
+        failure::bail!("failed to execute `{:?}`: exited with {}", command, status)
+    }
+}
 
-        bundle
-            .write(&wranglerjs_output)
-            .expect("could not write bundle to disk");
+pub fn run_build_and_watch(
+    project: &Project,
+    tx: Option<Sender<()>>,
+) -> Result<(), failure::Error> {
+    let (mut command, temp_file, bundle) = setup_build(project)?;
+    command.arg("--watch=1");
+
+    info!("Running {:?} in watch mode", command);
+
+    thread::spawn(move || {
+        let _command_guard = util::GuardedCommand::spawn(command);
+
+        let (watcher_tx, watcher_rx) = channel();
+        let mut watcher = watcher(watcher_tx, Duration::from_secs(1)).unwrap();
```

Comment:
i am worried about these unwraps!

Reply:
if we can: let's get rid of unwraps- make them errors if at all possible. if not possible: info debugging lines would be very useful. ask yourself- when this panics.. what am i gonna want to know

Reply:
(this is basically creating an `expect`)